pages/linux/scrun.md

```diff
@@ -0,0 +1,24 @@
+# scrun
+
+> An OCI (Open Container Initiative) runtime proxy for Slurm.
+> More information: <https://slurm.schedmd.com/scrun.html>.
+
+- Create a container in the current directory:
+
+`scrun create {{container_id}}`
+
+- Start a container in a Slurm job:
+
+`scrun start {{container_id}}`
```

Comment:
Is there a way to pass arguments to a container at all? It might be worth having an extra example directly beneath this one about that if so.